extensions/publisher-command-center/src/components/Collaborators.js

```diff
@@ -0,0 +1,24 @@
+import m from "mithril";
+
+import Content from "../models/Content";
+
+export default {
+  view: function () {
+    const content = Content.data;
+    const accessUrl = content?.dashboard_url ? `${content.dashboard_url}/access` : "#";
+
+    return m(".pt-3.border-top", [
+      m(".", [
+        m("h5", "Collaborators"),
+        m(
+          "p",
+          m("small.text-body-secondary.align-items-center", [
+            m(".fa-solid.fa-user-lock"),
+            " ",
+            m("a", { href: accessUrl, target: "_blank" }, "Manage Access"),
```

Comment:
```suggestion
            m("a", { href: accessUrl, target: "_blank" }, "See Collaborators and Manage Access"),
```

Is this too much?

Reply:
I don't think so. I just tried it out and looks good 👌 .
![CleanShot 2025-05-15 at 14 14 05](https://github.com/user-attachments/assets/5013d7d0-19ea-497d-ae76-f685914c77ab)